<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1160</width>
    <height>707</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">width:100%;</string>
   </property>
   <widget class="QGroupBox" name="main_layout">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1161</width>
      <height>711</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(193, 255, 243);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>360</y>
       <width>651</width>
       <height>151</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QPushButton" name="test_button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>19</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 170, 255);</string>
        </property>
        <property name="text">
         <string>Тестирование</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="result_button">
        <property name="font">
         <font>
          <pointsize>19</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 170, 255);</string>
        </property>
        <property name="text">
         <string>Просмотр результатов</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="test_menu">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1171</width>
      <height>721</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>999999</width>
      <height>99999</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(73, 97, 255);
width:100%;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>110</y>
       <width>511</width>
       <height>431</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="dezh_po_chasti">
        <property name="minimumSize">
         <size>
          <width>309</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 170, 255);</string>
        </property>
        <property name="text">
         <string>Дежурный по части</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pom_dezh_po_chasti">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 170, 255);</string>
        </property>
        <property name="text">
         <string>Помошник дежурного</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="back_to_main_window">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(23, 255, 166);</string>
        </property>
        <property name="text">
         <string>Назад</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="test_box">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1161</width>
      <height>711</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 170, 127);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="question_text">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>1061</width>
       <height>131</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>1061</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1061</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(85, 255, 0);</string>
     </property>
     <property name="text">
      <string>Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc </string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>190</y>
       <width>1054</width>
       <height>461</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="question_list_layout">
      <item>
       <layout class="QHBoxLayout" name="question_box_1">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QLabel" name="answer_var_1">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>950</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>950</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="text">
           <string>Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc v</string>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="choose_answer_1">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>44</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="question_box_2">
        <item>
         <widget class="QLabel" name="answer_var_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>950</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>950</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="text">
           <string>Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc v</string>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="choose_answer_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>44</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="question_box_3">
        <item>
         <widget class="QLabel" name="answer_var_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>950</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>950</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="text">
           <string>Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc v</string>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="choose_answer_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>44</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="question_box_4">
        <item>
         <widget class="QLabel" name="answer_var_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>950</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>950</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="text">
           <string>Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc Djcghjc v</string>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="choose_answer_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>44</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="next_question">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>660</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 170, 255);</string>
     </property>
     <property name="text">
      <string>Следующий вопрос</string>
     </property>
    </widget>
   </widget>
   <zorder>test_menu</zorder>
   <zorder>test_box</zorder>
   <zorder>main_layout</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
